doc/keepalived.conf.SYNOPSIS

```diff
@@ -579,6 +579,8 @@ virtual_server fwmark <INTEGER>    {  # VS fwmark declaration
 virtual_server group <STRING>      {  # VS group declaration
     ip_family inet|inet6  # Address family
     delay_loop <INTEGER>  # delay timer for service polling
+    lb_algo rr|[?]        # Which load-balancing algorithm to use?
```

Comment:
lvs_sched and lb_algo are synonyms, and lvs_sched is documented. lvs_sched is the preferred keyword, and so lb_algo is not documented.